frontend/lib/widgets/match_alert.dart

```diff
@@ -0,0 +1,139 @@
+                  ),
+                ],
+              ),
+            ),
+            isStudent
+                ? Container(
+                    padding: const EdgeInsets.all(40.0),
+                    child: Column(
+                      children: [
+                        Text(
+                          object.name + ' ' + object.surname,
+                          maxLines: 22,
+                          style: TextStyle(
+                            fontSize: 30,
+                            fontWeight: FontWeight.bold,
+                            color: Colors.white,
+                          ),
+                        ),
+                        Text(
+                          'School of Life',
+                          style: TextStyle(
+                            fontSize: 20.0,
+                            fontWeight: FontWeight.bold,
+                            color: Colors.white,
+                          ),
+                        )
+                      ],
+                    ),
+                  )
+                : Column(
+                    children: [
+                      Container(
+                        // Image container
+                        decoration: BoxDecoration(
+                            image: DecorationImage(
+                              image: ExactAssetImage("/images/image0.jpg"),
+                              fit: BoxFit.fill,
+                            ),
+                            border: Border.all(
+                              color: Colors.grey,
+                              width: 2.0,
+                            ),
+                            borderRadius: BorderRadius.circular(10.0)),
+                        width: size.width * .2,
+                        height: size.width * .2,
+                      ),
+                      Text(
+                        'Google, Zurich',
+                        maxLines: 20,
+                        style: TextStyle(
+                            fontSize: 30,
+                            fontWeight: FontWeight.bold,
+                            color: Colors.white),
+                      ),
+                    ],
+                  ),
+            Container(
+              padding: const EdgeInsets.all(40.0),
+              child: Column(
+                children: [
+                  Text(
+                    'for',
+                    style: TextStyle(
+                      fontSize: 18.0,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.white,
+                    ),
+                  ),
+                  Text(
+                    placement.position,
+                    style: TextStyle(
+                      fontSize: 40.0,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.white,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
```

Comment:
@AliceCasali maybe some spacing with MainAxisAlignment.spaceEvenly between the elements in the main Column could give a nicer feeling of the card, rn everything is a bit squished together